docs/source/test_network.md

```diff
@@ -20,9 +20,9 @@ current master branch, it is possible that you will encounter errors.
 
 ## Before you begin
 
-Before you can run the test network, you need to clone the `fabric-samples`
-repository and download the Fabric images. Make sure that you have installed
-the [Prerequisites](prereqs.html) and [Installed the Samples, Binaries and Docker Images](install.html).
+Before you can run the test network, you need to install Fabric Samples in your
+environment. Follow the instructions on [getting_started](getting_started.html)
+to install the required software.
```

Comment:
There is a reference in  test_network.md file to the "Windows extras" section of the prereqs which no longer exists.
I would suggest changing it from
This was most likely caused by not setting
   ``core.autocrlf`` to ``false`` in your Git configuration (see
    [Windows extras](prereqs.html#windows-extras)).
 to
 This was most likely caused by not setting
   ``core.autocrlf`` to ``false`` in your Git configuration (see
    [Windows](prereqs.html#windows)).

Reply:
Done.